### PR TITLE
fix: Set TENSORBOARD_PROXY_URL so TensorBoard can easily be used in JupyterLab

### PR DESCRIPTION
**Pull request by rushtehrani**
**What this PR does**:
This allows TensorBoard to be used in Onepanel's JupyterLab Workspaces.

**Special notes for your reviewer**:
Example usage in JupyterLab:
```
# Load the TensorBoard notebook extension.
%load_ext tensorboard

# Run TensorBoard within Onepanel's JupyterLab
%tensorboard --logdir <path-to-logs> --path_prefix /tensorboard
```

**Checklist**

Please check if applies

- [ ] I have added/updated relevant unit tests
- [ ] I have added/updated relevant documentation

Required 

- [x] I accept to release these changes under the Apache 2.0 License

### CODE REVIEW

File: db/go/20201031165106_add_tensorboard_env_var_to_jupyterlab_template.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		goose.AddMigration(Up20201031165106, Down20201031165106)
+		initializedMigrations[20201031165106] = true
+	}
+}
+
+// Up20201031165106 updates the jupyterlab workspace to include container lifecycle hooks.
+// These hooks will attempt to persist conda, pip, and jupyterlab extensions between pause and shut-down.
+func Up20201031165106(tx *sql.Tx) error {
+	// This code is executed when the migration is applied.
+	return updateWorkspaceTemplateManifest("20201031165106_jupyterlab.yaml", jupyterLabTemplateName)
```
Comment (rushtehrani) [inline db/go/20201031165106_add_tensorboard_env_var_to_jupyterlab_template.go:19]: @aleksandrmelnikov @Vafilor I think for the YAML name, we should have the timestamp first so things are ordered in the IDE accordingly. Thoughts?

Comment (Vafilor) [reply db/go/20201031165106_add_tensorboard_env_var_to_jupyterlab_template.go:19]: @rushtehrani Good point, yeah, let's do that.

File: db/go/20201031165106_add_tensorboard_env_var_to_jupyterlab_template.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+// Down20201031165106 removes the lifecycle hooks from the template.
+func Down20201031165106(tx *sql.Tx) error {
+	// This code is executed when the migration is rolled back.
+	return updateWorkspaceTemplateManifest("20201028145442_jupyterlab.yaml", jupyterLabTemplateName)
```
[the remark this answers is not in the document]
Comment (Vafilor) [reply db/go/20201031165106_add_tensorboard_env_var_to_jupyterlab_template.go:25]: I think that's fine.

File: db/go/20201031165106_add_tensorboard_env_var_to_jupyterlab_template.go (hunk 1 above)
Comment (aleksandrmelnikov) [reply db/go/20201031165106_add_tensorboard_env_var_to_jupyterlab_template.go:19]: Agreed

File: db/go/20201031165106_add_tensorboard_env_var_to_jupyterlab_template.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (aleksandrmelnikov) [reply db/go/20201031165106_add_tensorboard_env_var_to_jupyterlab_template.go:25]: Sure

**Pull request merged**